static/app/views/ddm/widgetDetails.tsx

```diff
@@ -15,6 +15,16 @@ enum Tab {
   CODE_LOCATIONS = 'codeLocations',
 }
 
+const selectedSeriesToQuery = (selectedSeries: string) => {
+  return (
+    selectedSeries
+      .split(', ')
+      // wrap everything after : in quotes
+      .map(series => series.replace(/(.*):(.*)/, '$1:"$2"'))
```

Comment:
We can have all chars except `.` and `\` in the quoted string, so this matching is unfortunately also ambiguous. The best solution I can think of, without having to parse properly the string, is to find the first occurrence of a `:` in each tag filter and split from there. Maybe the regex engine of js takes this route but it is not sure, given your regex.